packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -231,6 +231,10 @@ export type AddMediaOptions = {
   remoteMediaManagerConfig?: RemoteMediaManagerConfiguration; // applies only to multistream meetings
   bundlePolicy?: BundlePolicy; // applies only to multistream meetings
   allowMediaInLobby?: boolean; // allows adding media when in the lobby
+  sendVideo?: boolean;
+  receiveVideo?: boolean;
+  sendAudio?: boolean;
+  receiveAudio?: boolean;
```

Comment:
Can we add these options under a more granular object? Something like below?
This way it becomes clearer to the developer on what they do. It can get confusing as to why we have `audioEnabled` and these options too.
Please add comments as well similar to the other properties above
```suggestion
  additionalMediaOptions: {
    sendVideo?: boolean;
    receiveVideo?: boolean;
    sendAudio?: boolean;
    receiveAudio?: boolean;
  }
```